docs/release_notes/7.0.26-14.md

```diff
@@ -0,0 +1,54 @@
+# Percona Server for MongoDB 7.0.26-14 ({{date.7_0_26}})
+
+[Installation](../install/index.md){.md-button}
+[Upgrade from MongoDB Community](../install/upgrade-from-mongodb.md){.md-button}
+
+Percona Server for MongoDB 7.0.26-14 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB Community Edition.
+
+Percona Server for MongoDB 7.0.26-14 includes the improvements and bug fixes of [MongoDB 7.0.25 Community Edition](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.25---oct-2--2025) and [MongoDB 7.0.26 Community Edition](https://www.mongodb.com/docs/manual/release-notes/7.0/#upcoming). It supports protocols and drivers of both MongoDB Community 7.0.25 and MongoDB Community 7.0.26.
+
+## Release highlights
+
+This release provides the following features and improvements:
+
+### Sunsetting Pro Builds
+
+In line with our open‑source mission, we are officially sunsetting Pro Builds. All features that were previously included in Pro Builds are now accessible to everyone in our regular public builds.
+
+If you require a Pro Build for a specific version, your Percona representative is ready to guide you. Our goal is to ensure a smooth transition and continued success with Percona's open‑source solutions, enhanced with enterprise‑grade capabilities.
```

Comment:
By saying `for a specific version` do we mean `for previously released version` or something else?